react/react1/week1/todolist/src/App.js

```diff
@@ -0,0 +1,24 @@
+import "./App.css";
+import tasks from "./TodoTask";
+import TodoTaskList from "./TodoTaskList";
```

Comment:
Everything is working but I'd suggest you refactor a few things:
- Change the TodoTask to something like data.js
- Then, rename TodoTaskList to TodoTask  (singular name).
It can be important to use names that make sense, and since TodoTaskList contains information about a single todo item it doesn't make much sense to have List in its name.

Finally, if you want to, you can keep TodoTaskList as an extra layer, but mapping the tasks should be done inside TodoTaskList not in the App.js. So, the components structure would like more like:

```
App
  TodoTitle
  TodoTaskList
    TodoTask (map the tasks here and return a TodoTask component)
    ... 
    TodoTask
```